### PR TITLE
[Stalled] Route creation fixup

### PR DESCRIPTION
**Pull request by ginold**
related to #411 
Waiting for `duration_attributes` to be added to the API first or to be discussed with the association.

### CODE REVIEW

File: c2corg_ui/templates/route/edit.html [hunk 1]
```diff
@@ -289,8 +289,19 @@ <h2 class="heading show-phone"><span translate>numbers</span></h2>
...
             </div>
 
+            ## DIFFICULTIES DURATIONS
+            <div class="form-group half" ng-class="{ 'has-success': route.duration_difficulties}">
+              <label><span translate>duration_difficulties</span> (<span translate>days</span>)</span></label>
+               <select class="form-control" ng-model="route.duration_difficulties">
+                 <option value=""></option>
+                 <option value="5"><span>5min</span></option>
+                 <option value="60"><span>1h</span></option>
+                 <option value="360"><span>6h</span></option>
```
Comment (asaunier) [inline c2corg_ui/templates/route/edit.html]: Where do those values come from?

**Comment by asaunier** [comment]
If this ``duration_attributes`` is really needed (????) it would make more sense to add it to the API first.

In addition, the PR shows many changes in the route attributes CSS classes (``data`` => ``duration_difficulties``). Incorrect search and replace I guess! Please always have a look at the diff of your PR before creating it!

File: c2corg_ui/templates/route/edit.html (hunk 1 above)
Comment (ginold) [inline c2corg_ui/templates/route/edit.html]: line 44 of https://docs.google.com/spreadsheets/d/1p7-RakklDot9gFnciEGox75833zU0ZuC5pRAoLZKziY/edit#gid=1418510514

Comment (asaunier) [inline c2corg_ui/templates/route/edit.html]: ?
Line 44 describes the total duration of the route (in days). Not the difficulties duration (in minutes).

**Comment by asaunier** [comment]
I think you can close this PR and remove the branch: this attribute is not to be implemented.

File: c2corg_ui/templates/route/edit.html (hunk 1 above)
Comment (ginold) [inline c2corg_ui/templates/route/edit.html]: sorry, 45!

**Comment by ginold** [comment]
:+1:

**Pull request closed without merging**